adafruit_pyportal.py

```diff
@@ -154,7 +156,8 @@ def __init__(self, *, url=None, headers=None, json_path=None, regexp_path=None,
                  default_bg=0x000000, status_neopixel=None,
                  text_font=None, text_position=None, text_color=0x808080,
                  text_wrap=False, text_maxlen=0, text_transform=None,
-                 image_json_path=None, image_resize=None, image_position=None,
+                 json_transforms=None, image_json_path=None,
```

Comment:
please make this `json_transform` to match the non-pluralization of `text_transform` :)
otherwise looks good!

Reply:
It's now singular.